Cargo.toml

```
@@ -18,3 +18,11 @@ name = "kairos"
 version = "0.1.0"
 edition = "2021"
 license = "MIT OR Apache-2.0"
+
+[workspace.dependencies]
+casper-client = "2.0"
+casper-types = "4.0"
+casper-client-types = { package = "casper-types", version = "3.0" }
+casper-client-hashing = { package = "casper-hashing", version = "2.0" }
+casper-contract = "4.0"
```

Please add missing `default-features = false`, as features as additive and that's causing build to fail:

```suggestion
casper-contract = { version = "4.0", default-features = false }
```